packages/lit-dev-content/site/docs/v3/tools/testing.md

```diff
@@ -75,3 +75,48 @@ export default {
 };
 ```
 
+## Using WebdriverIO
+
+[WebdriverIO](https://webdriver.io) is a good alternative for your component or end-to-end tests. It has very compelling advantages like support for [mocking](https://webdriver.io/docs/component-testing/mocking) and [code coverage](https://webdriver.io/docs/component-testing/coverage) reporting.
+
+You can set up WebdriverIO in your project via:
+
+```bash
+npm init wdio@latest ./
+```
+
+It will start a configuration wizard that will guide you through some questions. Make sure select the following:
+
+- __What type of testing would you like to do?__:<br />Component or Unit Testing - in the browser
+- __Which framework do you use for building components?__:<br />Lit
+
+The remaining questions can be answered as desired.
+
+In order test the component you have to render it into the test page before the test starts and ensure it gets cleaned up afterwards:
+
+```ts
+import { expect, $ } from '@wdio/globals'
+
+// import simple-greeting component from https://lit.dev/docs/components/overview/
```

Comment:
May be worth slightly expanding this comment. The lowercase `import` at the start of this line made me initially think this was a commented out import.

```suggestion
// Component.ts contains the <simple-greeting> component implemented the same as:
// https://lit.dev/docs/components/overview/
```